presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -53,6 +53,8 @@ Description
 
 Retrieve rows from zero or more tables.
 
+.. contents::
```

Comment:
it's a bit strange that TOC contains likes to some sections above it.

I'd consider putting it at the top, or configuring it to strip the sections that are above this place.

@mosabua @electrum WDYT?

Reply:
Having it at the very top pushed the most important part (the syntax and description) down below the first page fold. Imho that should be avoided. Once you read the first bit .. you see the ToC that gives the reader an idea what else there is. Just my 2c though .. I am fine with moving it to the top... we should however ensure it is somewhere on the page since there is no other easy way to see all the useful info on that page as a short version.

Reply:
I like the TOC. And yes, having it at the top might be overwhelming.
Is there a way to strip the sections that are above?

(if not, i'm fine with how it is now)

Reply:
There does not seem to be a way to strip sections..